crawlers/src/delays/index.ts

```diff
@@ -169,7 +178,9 @@ async function insertDepartureDelaysIntoDb(
         stop.departureTimeEstimated,
         rawData,
         vrrTimetableVersionId,
-        stop.transportation.properties?.tripCode
+        stop.transportation.properties?.tripCode,
+        parentStopId,
```

Comment:
Why I am asking what ids are crawled with (see question on this file), the format being used for stop_id is relevant for later queries on delay data as the crawled id is written into delay data as parentStopId.

Reply:
When planning a journey, the format of origin/destination would be e.g. 'de:05111:18235:90:4' (for "Düsseldorf Hbf"). As we don´t want to depend on a specific plattform, parent stop_id should be used to find delay data; in this case this would be 'de:05111:18235'. **Question is, is it matching with what gets written into delay data table?**

Reply:
> Question is, is it matching with what gets written into delay data table?

I would hope so. We will see tomorrow after running the crawler for a while.